dev-python/neovim-remote/neovim-remote-1.8.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_{4,5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="A tool that helps control neovim processes"
+HOMEPAGE="https://github.com/mhinz/neovim-remote"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/mhinz/neovim-remote.git"
+else
+	SRC_URI="https://github.com/mhinz/neovim-remote/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
```

Comment:
will it run on x86 too? You can substitute neovim-remote in SRC_URI with a variable

Reply:
It will run on x86, but neovim is keyworded "~amd64" so I didn't add it to the list.

Yeah, alright. I'll add the substitutions.